### PR TITLE
Styling Changes

### PR DESCRIPTION
**Pull request by KayleeLChan**
Configurated custom tailwind assets, restyled the main page, analysis page, intent diagrams, base button, and modal to be consistent with our ux design.

ToDo:
- [x] Fix intent diagram resizing in analysis page
- [x] Different base button sizes
- [ ] svg in BestFlow title

### Change types

- [x] Non-breaking change
- [x] Front end change

### CODE REVIEW

File: front-end/src/components/AnalysisPage.jsx [hunk 1]
```diff
@@ -6,24 +6,24 @@ import IntentLister from './IntentLister.jsx';
 
 const fakeIntents = [
   {
-    question: "hi",
+    question: "What is something that Waluigi and Wario often say?",
     children: {
-      "q1": 100,
+      "The brothers Waluigi and Wario have their catchphrase 'waaa'.": 100,
     }
```
Comment (WeissSa) [inline front-end/src/components/AnalysisPage.jsx:12]: Thank u for keeping this.

File: front-end/src/components/BaseButton.jsx [hunk 2]
```diff
@@ -4,10 +4,27 @@ import PropTypes from 'prop-types';
...
+        //Styling for size presets
+        switch(props.size) {
+            case "sm":
+                styling += "py-1 px-4 md:text-md 2xl:text-lg";
+                break;
+            case "lg":
+                styling += "md:py-5 md:px-6 md:text-2xl 2xl:py-6 2xl:px-9 2xl:text-2xl";
+                break;
+            default:
+                styling += "py-3 px-6 md:text-lg 2xl:text-2xl";
```
Comment (WeissSa) [inline front-end/src/components/BaseButton.jsx:24]: What is the intended default case here (maybe add a comment)

File: front-end/src/components/IntentDiagram.jsx [hunk 3]
```diff
@@ -6,7 +6,9 @@ class IntentDiagram extends Component {
         return Object.keys(this.props.branches).map((key) => {
             return (
                 <div 
-                    className="text-sm bg-emerald-500 flex-col flex gap-y-0 shadow-sm rounded-2xl p-5 border border-emerald-500 hover:border-black focus-within:border-black"
+                    className="container object-contain mx-auto text-md place-content-center
+                    bg-green-100 flex-col flex gap-y-0 shadow-md shadow-blue/10 rounded-2xl p-5
+                    border-2 border-green-200 hover:border-black focus-within:border-black"
```
Comment (WeissSa) [inline front-end/src/components/IntentDiagram.jsx:11]: :fire:

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 4]
```diff
@@ -36,26 +35,30 @@ class TranscriptUploadModal extends React.Component {
...
                 name="transcript"
                 data-testid="fileInput"
                 onChange={(e)=>this.handleFile(e)}>
               </input>
-              <BaseButton
-                data-testid="uploadButton"
-                click={(e)=>this.handleUpload(e)}
-                text="Upload"
-              />           
+              <div className={"justify-center flex"}>
```
Comment (WeissSa) [inline front-end/src/components/TranscriptUploadModal.jsx:54]: Curly braces arent needed here

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 5]
```diff
@@ -36,26 +35,30 @@ class TranscriptUploadModal extends React.Component {
...
         shouldCloseOnEsc={true}
         ariaHideApp={false}
       >
-        <h2 className="justify-center flex m-3" data-testid="upload-transcript-modal">
+        <h2 className="justify-center flex mt-3 text-lg" data-testid="upload-transcript-modal">
           Drag and drop file or upload below.
         </h2>
           <div className={"justify-center flex m-7 flex-col"}>
               <input
+                className={"mb-7 ml-7"}
```
Comment (WeissSa) [inline front-end/src/components/TranscriptUploadModal.jsx:48]: +1

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 6]
```diff
@@ -36,26 +35,30 @@ class TranscriptUploadModal extends React.Component {
...
+        className="container w-60 md:w-80 mx-auto font-cabin bg-purple-100 rounded-lg shadow-lg py-3 mt-[40vh]"
         onRequestClose={this.props.toggleModal}
         shouldCloseOnEsc={true}
         ariaHideApp={false}
       >
-        <h2 className="justify-center flex m-3" data-testid="upload-transcript-modal">
+        <h2 className="justify-center flex mt-3 text-lg" data-testid="upload-transcript-modal">
           Drag and drop file or upload below.
         </h2>
           <div className={"justify-center flex m-7 flex-col"}>
```
Comment (WeissSa) [inline front-end/src/components/TranscriptUploadModal.jsx:46]: +1

File: front-end/tailwind.config.js [hunk 7]
```diff
@@ -1,10 +1,51 @@
...
+        DEFAULT: '2px 2px 2px var(--tw-shadow-color)',
+        lg: '0 8px 16px var(--tw-shadow-color)',
+      },
+    },
   },
-  plugins: [],
+  plugins: [
+    plugin(function ({ matchUtilities, theme }) {
+      matchUtilities(
+        {
```
Comment (WeissSa) [inline front-end/tailwind.config.js:42]: Sheeesh

**Review by WeissSa — CHANGES_REQUESTED** [review]
Some small changes + what we discussed on discord. Really good work :fire:

**Review by williamyao27 — CHANGES_REQUESTED** [review]
Nitpick comment and also add some testing for new props in BaseButton. But otherwise LGTM! Great job rewriting getStyle and on styling the FE.

File: front-end/src/components/AnalysisPage.jsx (hunk 1 above)
Comment (KayleeLChan) [reply front-end/src/components/AnalysisPage.jsx:12]: waa

Comment (WeissSa) [reply front-end/src/components/AnalysisPage.jsx:12]: waaa

**Review by williamyao27 — APPROVED** [review]
LETS GO TEAM

**Pull request merged**